src/screens/DataFilter.js

```diff
@@ -56,10 +56,11 @@ const DataFilterPage = () => (
 
         <Property name="range">
           <Description>
-            The range of numeric values. Should not be used with 'options'.
+            The range of numeric values. Should not be used with 'options'. If
+            not specified, the default step is 20.
```

Comment:
Isn't the default step the total range divided by 20?

Reply:
You're right, let me adjust the wording.

Reply:
Adjusted this wording.